bin/hipconvertinplace.sh

```diff
@@ -6,7 +6,7 @@
 # This can be quite handy when dealing with an existing CUDA code base since the script
 # preserves the existing directory structure.
 
-SCRIPT_DIR=`dirname $0`
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
```

Comment:
The manual page for readlink suggests using realpath is preferred.

Reply:
Thanks Icarus. Updated the patch by using realpath for getting the absolute path